src/modules/launcher/Microsoft.Launcher/dllmain.cpp

```diff
@@ -202,9 +202,16 @@ class Microsoft_Launcher : public PowertoyModuleIface
                 sei.lpFile = L"modules\\launcher\\PowerLauncher.exe";
                 sei.nShow = SW_SHOWNORMAL;
                 sei.lpParameters = executable_args.data();
-                ShellExecuteExW(&sei);
-
-                m_hProcess = sei.hProcess;
+                
+                if (ShellExecuteExW(&sei))
+                {
+                    m_enabled = true;
+                    m_hProcess = sei.hProcess;
+                }
+                else
+                {
+                    Logger::error("Launcher failed to start");
```

Comment:
```suggestion
                    Logger::error("PT Run failed to start");
```

Reply:
Sure, why not, but we have a bunch of log entries in this file where it's called "Launcher", do I change those as well?

Reply:
We should progressively remove the term "launcher" from everywhere.